skins/laika/src/components/pages/UpdateAddress.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+	<div class="address-update-form">
+		<form name="address-update" ref="form" :action="updateAddressURL" method="post">
+			<h1 class="title is-size-1">{{ $t('address_change_form_title') }}</h1>
+			<legend class="title is-size-5">{{ $t( 'address_change_form_label' ) }}</legend>
+			<div>
+				<name :show-error="fieldErrors"
+						:form-data="formData"
+						:address-type="addressType"
+						v-on:field-changed="onFieldChange">
+				</name>
+				<postal :show-error="fieldErrors"
+						:form-data="formData"
+						:countries="countries"
+						v-on:field-changed="onFieldChange">
+				</postal>
+			</div>
+			<div class="level has-margin-top-36">
+				<div class="level-right">
+					<b-button id="next" :class="['is-form-input-width', $store.getters.isValidating ? 'is-loading' : '', 'level-item']"
+							  @click.prevent="submit()"
+							  type="is-primary is-main">
+						{{ $t('address_change_form_submit') }}
+					</b-button>
+				</div>
+			</div>
+		</form>
+	</div>
+</template>
+<script lang="ts">
+import Vue from 'vue';
+import Name from '@/components/shared/Name.vue';
+import Postal from '@/components/shared/Postal.vue';
+import { AddressValidity, AddressFormData, ValidationResult } from '@/view_models/Address';
+import { Validity } from '@/view_models/Validity';
+import { NS_ADDRESS } from '@/store/namespaces';
+import { setAddressField, validateAddress, setReceiptOptOut, setAddressType } from '@/store/address/actionTypes';
+import { action } from '@/store/util';
+import { AddressTypeModel } from "@/view_models/AddressTypeModel";
+import { mapGetters } from "vuex";
+
+export default Vue.extend( {
+	name: 'update-address',
```

Comment:
Oh I think we should stick to the naming convention we have for Vue components in laika. So that would be `UpdateAddress`
```suggestion
	name: 'UpdateAddress',
```